wiki/Tournaments/Official_support/en.md

```diff
@@ -90,6 +91,17 @@ Once the tournament has concluded, the tournament organisers will need to submit
 - An appropriately designed badge ([see below](#profile-badges)).
 - A link to the publicly viewable catalogue of your tournament's match history.
 
+#### Referee bots/automation
+
+The use of bots/automation to aid in the running of officially supported tournament matches is permitted *with some caveats to reflect their cutting-edge nature*, so long as any applicable software/workflows meets the requirements listed below:
+
+- The bot/automation must be made publicly and functionally available as an open-source repository.
+- The bot/automation must be attached to and run from a singular [Bot account](/wiki/Bot_account), following all stipulations and requirements listed therein.
+- If a bot/automation is used for refereeing, a human referee must always be available to respond to calls for support from participants within a reasonable timeframe (2-5 minutes), and able to be summoned remotely via the use of a chat command inside the lobby itself (we recommend `!panic` as a global alias for this).
```

Comment:
this states 2-5 minutes, but the bullet in the section above states <2 minutes. Should probably pick one or the other and be consistent.